clients/cli/docs/README.md

```diff
@@ -76,46 +76,60 @@ To run the Gravitino CLI, use the following command structure:
 ```bash
 usage: gcli [metalake|catalog|schema|table] [list|details|create|delete|update] [options]
 Options
- -C,--create       create an entity
- -D,--details      list details about an entity
  -f,--name <arg>   full entity name (dot separated)
  -h,--help         command help information
  -L,--list         list entity children
```

Comment:
the "list" is a command, should it be removed from options as well?

Reply:
you are looking at an outdated version of README

Reply:
It was missing metalake so I've added that